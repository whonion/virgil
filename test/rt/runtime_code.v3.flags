-runtime-code-size=4K
