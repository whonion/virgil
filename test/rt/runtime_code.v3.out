##+code size check
##-ok
##+generating code
##-ok
##+checking code
##-ok
##+running code
##-ok
